start/sw/gui-app/qt.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Qt
+subsection: gui-app
+section: start-sw
+description: 
+---
+
+# Create an application using Qt
+
+[Qt][0] is a cross platform application framework which contains a cross platform
+graphical toolkit. Unlike GTK+ which is developed mostly by Linux users, Qt is
+developed by The Qt Company. Because of that Qt have better cross platform support.
```

Comment:
Could you at least put how to install QT here / or simply put here whether it's in Fedora.

Reply:
I don't know how to install qt myself so I can't explain it.

Reply:
Then please do not create the page if you haven't tested it.

Reply:
We only want the content someone has tested on Fedora. See [our wiki](https://fedoraproject.org/wiki/Websites/Developer).